docs/docs/sourcing-from-drupal.md

```diff
@@ -2,7 +2,48 @@
 title: Sourcing from Drupal
 ---
 
-This is a stub. Help our community expand it.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+### Why use Drupal + Gatsby together?
+
+Using Drupal as a headless CMS with Gatsby is a great way to get an enterprise-quality CMS for free, paired with a great modern development experience and all the benefits of the JAMstack, like performance, scalability, and security.
+
+It only takes a few steps to use Gatsby with Drupal as a headless CMS (also known as decoupled Drupal).
+
+<a href="https://using-drupal.gatsbyjs.org/">A delicious Gatsby + Drupal site</a>
```

Comment:
It's better if you use the markdown notation for links:
`[name](link)`

So that remark plugins can catch those and internal links get the benefits of Gatsby's internal Link component.

Same applies to the rest of the links of course.

Reply:
Sure! Will do that! Gimme a few minutes. @LeKoArts

Reply:
So is everything else okay?

Reply:
That's cool. All the changes requested have been made!